object-oriented-programming/blackjack-game/python/hand_class.py

```diff
@@ -0,0 +1,39 @@
+# Isaac Computer Science
+# Usage licensed under the Open Government Licence v3.0
+# BETA VERSION
+
+class Hand:
+
+    def __init__(self):
+        self.__cards = []
+        self.__value = 0
+        self.__aces = 0
+
+    def add_card(self, card):
+        self.__cards.append(card)
+        if card.get_rank() == "Ace":
+            self.__aces += 1
+        self.__value += card.get_value()
+        if self.__value > 21: # adjust for aces
```

Comment:
I think this needs some more explanation as to what you are doing with regard to the value of the aces

Reply:
Also, what happens in the eventuality that you originally wanted an ace to be 11 and then after twisting you actually want it to be 1?

Reply:
That's ok as it re-evaluates each time nd makes best choice.